### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by MaximAlien**
### Description

PR updates dependencies to:
- Mapbox Maps `v10.7.0-beta.1`
- Navigation Native `v108.0.0`
- Mapbox Common `v22.1.0-beta.1`
- Mapbox Speech `v2.0.1`

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" ~> 22.0
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 106.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 22.0.0-rc.2
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 108.0
```
Comment (MaximAlien) [inline Cartfile:2]: After upgrade crash occurs after finishing active navigation:

Screen Shot 2022-06-30 at 10 17 49 AM

File: Sources/MapboxNavigation/NavigationMapView.swift [hunk 2]
```diff
@@ -1939,7 +1939,7 @@ open class NavigationMapView: UIView {
         mapView.ornaments.options.scaleBar.visibility = .hidden
         storeLocationProviderBeforeSimulation()
         
-        mapView.mapboxMap.onEvery(.renderFrameFinished) { [weak self] _ in
+        mapView.mapboxMap.onEvery(event: .renderFrameFinished) { [weak self] _ in
```
Comment (bamx23) [inline Sources/MapboxNavigation/NavigationMapView.swift:1942]: Do we have breaking changes in Maps SDK?

**Review by bamx23 — APPROVED** [review]

File: Sources/MapboxNavigation/NavigationMapView.swift (hunk 2 above)
Comment (MaximAlien) [reply Sources/MapboxNavigation/NavigationMapView.swift:1942]: Not breaking change, but deprecation in this cases. Modified signature to fix warnings.

Comment (bamx23) [reply Sources/MapboxNavigation/NavigationMapView.swift:1942]: Perfect, thanks

**Review by ShanMa1991 — APPROVED** [review]
Maps [v10.7.0-beta.1](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.7.0-beta.1) contains the fix of the `mapView` rendering issue when going from the background to the foreground. Should we mention this fix in the changeLog?

**Pull request merged**